src/redux/services/fileEntry.ts

```diff
@@ -530,17 +531,8 @@ export function removePath(absolutePath: string, state: AppState, fileEntry: Fil
     }
   }
 
-  reprocessResources(
```

Comment:
@devcatalin @erdkse removed this since I couldn't think of anything that needs to be reprocessed except for refs that are removed by the line below instead